Upstream-Status: Submitted
Signed-off-by: Ross Burton <ross.burton@intel.com>

From ca05ee2fb6db5b3b1edc16dba6150a2c6078a6cf Mon Sep 17 00:00:00 2001
From: Ross Burton <ross.burton@intel.com>
Date: Tue, 9 May 2017 15:30:00 +0100
Subject: [PATCH] Makefile: ensure utils/ exists before writing to it

Since utils/ was changed to not build recursively it's possible that in
out-of-tree builds the utils/ directory hasn't been created when Make runs the
utils/kconfig-tweak target which tries to write to file inside utils/.

To ensure this can work, mkdir the directory.

Signed-off-by: Ross Burton <ross.burton@intel.com>
---
 Makefile.am | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Makefile.am b/Makefile.am
index c8e96ef..1baa110 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -348,6 +348,7 @@ EXTRA_DIST += \
 	utils/kconfig-tweak.in.patch
 
 utils/kconfig-tweak: utils/kconfig-tweak.in
+	$(MKDIR_P) $(@D)
 	$(AM_V_GEN)$(SED) -e "s/@CONFIG_@/$(config_prefix)/g" \
 		$< >$@
 	@chmod +x $@
-- 
2.8.1
